# SNP23: Analyzing the genomic differences between wild type strains (as published in reference genomes) and strains kept under lab conditions (resequenced by the Segrè lab in 2019)

## ❓Questions we are trying to answer:
* What are the SNPs in the re-sequenced genomes (vs original genomes? I.e., how many SNAPs/genome, location of SNPs, what genes are affected? 
* What are other genomic changes compared with the original genome? E.g, rearrangements, paralogs.
* Is there conservancy in the number of SNPs based on phylogenetic relationships? I.e Do closely-related microbes undergo similar changes?

## 🔗 Useful Links:
Shared Notes on Google Drive: https://drive.google.com/drive/folders/1ZNowof5NB3n_OAjHu4QRoAtlrP9T6WHK?usp=drive_link

Here is a simple flow chart:

```mermaid
graph TD;
    A-->B;
    A-->C;
    B-->D;
    C-->D;
```

