# SNP23: Analyzing the genomic differences between wild type strains (as published in reference genomes) and strains kept under lab conditions (resequenced by the Segr√® lab in 2019)

## ‚ùìQuestions we are trying to answer:
* What are the SNPs in the re-sequenced genomes (vs original genomes? I.e., how many SNAPs/genome, location of SNPs, what genes are affected? 
* What are other genomic changes compared with the original genome? E.g, rearrangements, paralogs.
* Is there conservancy in the number of SNPs based on phylogenetic relationships? I.e Do closely-related microbes undergo similar changes?

## üîó Useful Links:
Shared Notes on Google Drive: https://drive.google.com/drive/folders/1ZNowof5NB3n_OAjHu4QRoAtlrP9T6WHK?usp=drive_link

## For the Boston University Community: Running on the BU SCC
Breseq is insalled on the BU SCC, along with all of the dependencies to run this pipeline. To activate the conda environment with these tools, run:
```
module load miniconda
conda activate /projectnb/hfsp/SNP23/envs/brese
```
