<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.36.0&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/TKDHS4_8_S40_R1_001.error_rates.pdf">errors</a></td><td>TKDHS4_8_S40_R1_001</td><td align="right">3,302,321</td><td align="right">525,069,039</td><td align="right">100.0%</td><td align="right">159.0&nbsp;bases</td><td align="right">159&nbsp;bases</td><td align="right">97.8%</td></tr><tr ><td><a href="calibration/TKDHS4_8_S40_R2_001.error_rates.pdf">errors</a></td><td>TKDHS4_8_S40_R2_001</td><td align="right">3,287,683</td><td align="right">522,741,597</td><td align="right">99.6%</td><td align="right">159.0&nbsp;bases</td><td align="right">159&nbsp;bases</td><td align="right">96.2%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>6,590,004</b></td><td align="right"><b>1,047,810,636</b></td><td align="right">99.8%</td><td align="right">159.0&nbsp;bases</td><td align="right"><b>159</b>&nbsp;bases</td><td align="right">97.0%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit dispersion</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/GCA_000511385.CP0069663776653.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>GCA_000511385.CP0069663776653</td><td align="right">3,776,653</td><td align="center">221.5</td><td align="center">5.8</td><td align="center">81.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/GCA_000511385.CP006967255493.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>GCA_000511385.CP006967255493</td><td align="right">255,493</td><td align="center">242.7</td><td align="center">4.3</td><td align="center">6.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/GCA_000511385.CP006968133621.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>GCA_000511385.CP006968133621</td><td align="right">133,621</td><td align="center">239.7</td><td align="center">6.6</td><td align="center">3.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/GCA_000511385.CP006969109815.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>GCA_000511385.CP006969109815</td><td align="right">109,815</td><td align="center">240.2</td><td align="center">4.2</td><td align="center">2.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/GCA_000511385.CP006970.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>GCA_000511385.CP006970</td><td align="right">77,876</td><td align="center">155.7</td><td align="center">3.8</td><td align="center">1.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/GCA_000511385.CP006971.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>GCA_000511385.CP006971</td><td align="right">77,775</td><td align="center">442.8</td><td align="center">19.8</td><td align="center">3.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/GCA_000511385.CP006972.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>GCA_000511385.CP006972</td><td align="right">68,752</td><td align="center">257.3</td><td align="center">5.3</td><td align="center">1.7%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/GCA_000511385.CP006973.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>GCA_000511385.CP006973</td><td align="right">40,170</td><td align="center">205.5</td><td align="center">4.5</td><td align="center">0.8%</td><td>Genus species strain strain.</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>4,540,155</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit dispersion</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">9329</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;2</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">220</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.016</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>GCA_000511385.CP0069663776653</td><td align="center">0.72779</td></tr><tr >
<td>GCA_000511385.CP006967255493</td><td align="center">0.71072</td></tr><tr >
<td>GCA_000511385.CP006968133621</td><td align="center">0.71479</td></tr><tr >
<td>GCA_000511385.CP006969109815</td><td align="center">0.71498</td></tr><tr >
<td>GCA_000511385.CP006970</td><td align="center">0.77469</td></tr><tr >
<td>GCA_000511385.CP006971</td><td align="center">0.63758</td></tr><tr >
<td>GCA_000511385.CP006972</td><td align="center">0.70002</td></tr><tr >
<td>GCA_000511385.CP006973</td><td align="center">0.73820</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.4.1</td></tr>
<tr><td>R</td><td>4.3.1</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>12:56:22&nbsp;24&nbsp;Jul&nbsp;2025</td><td>12:57:47&nbsp;24&nbsp;Jul&nbsp;2025</td><td>1&nbsp;minute&nbsp;25&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>12:57:47&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:21:27&nbsp;24&nbsp;Jul&nbsp;2025</td><td>23&nbsp;minutes&nbsp;40&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>13:21:27&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:22:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>1&nbsp;minute&nbsp;21&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>13:22:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:26:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>4&nbsp;minutes&nbsp;0&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>13:26:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:26:50&nbsp;24&nbsp;Jul&nbsp;2025</td><td>2&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>13:26:50&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:32:28&nbsp;24&nbsp;Jul&nbsp;2025</td><td>5&nbsp;minutes&nbsp;38&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>13:32:28&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:34:25&nbsp;24&nbsp;Jul&nbsp;2025</td><td>1&nbsp;minute&nbsp;57&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>13:34:25&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:37:46&nbsp;24&nbsp;Jul&nbsp;2025</td><td>3&nbsp;minutes&nbsp;21&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>13:37:46&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:39:29&nbsp;24&nbsp;Jul&nbsp;2025</td><td>1&nbsp;minute&nbsp;43&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>13:39:29&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:39:34&nbsp;24&nbsp;Jul&nbsp;2025</td><td>5&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>13:39:34&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:56:45&nbsp;24&nbsp;Jul&nbsp;2025</td><td>17&nbsp;minutes&nbsp;11&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>13:56:45&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:56:46&nbsp;24&nbsp;Jul&nbsp;2025</td><td>1&nbsp;second</td></tr>
<tr><td>Output</td><td>13:56:46&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:57:14&nbsp;24&nbsp;Jul&nbsp;2025</td><td>28&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>13:56:46&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:56:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>2&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>13:56:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:56:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>0&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>1&nbsp;hour&nbsp;54&nbsp;seconds</b></td>
</tr>
</table></body></html>