<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.36.0&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/B9NRXP_11_S115_R1_001.error_rates.pdf">errors</a></td><td>B9NRXP_11_S115_R1_001</td><td align="right">1,306,971</td><td align="right">207,808,389</td><td align="right">100.0%</td><td align="right">159.0&nbsp;bases</td><td align="right">159&nbsp;bases</td><td align="right">97.6%</td></tr><tr ><td><a href="calibration/B9NRXP_11_S115_R2_001.error_rates.pdf">errors</a></td><td>B9NRXP_11_S115_R2_001</td><td align="right">1,302,092</td><td align="right">207,032,628</td><td align="right">99.6%</td><td align="right">159.0&nbsp;bases</td><td align="right">159&nbsp;bases</td><td align="right">96.8%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>2,609,063</b></td><td align="right"><b>414,841,017</b></td><td align="right">99.8%</td><td align="right">159.0&nbsp;bases</td><td align="right"><b>159</b>&nbsp;bases</td><td align="right">97.2%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit dispersion</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/A37QDRAFT_scaffold_0.1282200.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_0.1282200</td><td align="right">282,200</td><td align="center">127.3</td><td align="center">2.2</td><td align="center">8.9%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_1.2281151.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_1.2281151</td><td align="right">281,151</td><td align="center">124.2</td><td align="center">2.1</td><td align="center">8.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_2.3252644.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_2.3252644</td><td align="right">252,644</td><td align="center">130.2</td><td align="center">2.1</td><td align="center">8.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_3.4207228.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_3.4207228</td><td align="right">207,228</td><td align="center">128.4</td><td align="center">2.3</td><td align="center">6.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_4.5202067.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_4.5202067</td><td align="right">202,067</td><td align="center">124.3</td><td align="center">2.3</td><td align="center">6.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_5.6181296.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_5.6181296</td><td align="right">181,296</td><td align="center">125.0</td><td align="center">2.0</td><td align="center">5.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_6.7175206.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_6.7175206</td><td align="right">175,206</td><td align="center">120.8</td><td align="center">2.1</td><td align="center">5.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_7.8155932.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_7.8155932</td><td align="right">155,932</td><td align="center">135.9</td><td align="center">2.2</td><td align="center">5.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_8.9138458.overview.png">coverage</a></td><td><a href="calibration/8.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_8.9138458</td><td align="right">138,458</td><td align="center">120.9</td><td align="center">2.2</td><td align="center">4.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_9.10119140.overview.png">coverage</a></td><td><a href="calibration/9.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_9.10119140</td><td align="right">119,140</td><td align="center">132.7</td><td align="center">2.3</td><td align="center">3.9%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_10.1198990.overview.png">coverage</a></td><td><a href="calibration/10.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_10.1198990</td><td align="right">98,990</td><td align="center">131.1</td><td align="center">2.3</td><td align="center">3.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_11.1287663.overview.png">coverage</a></td><td><a href="calibration/11.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_11.1287663</td><td align="right">87,663</td><td align="center">125.1</td><td align="center">2.2</td><td align="center">2.9%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_12.1382347.overview.png">coverage</a></td><td><a href="calibration/12.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_12.1382347</td><td align="right">82,347</td><td align="center">129.7</td><td align="center">2.1</td><td align="center">2.7%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_13.1479288.overview.png">coverage</a></td><td><a href="calibration/13.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_13.1479288</td><td align="right">79,288</td><td align="center">119.7</td><td align="center">1.9</td><td align="center">2.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_14.1576687.overview.png">coverage</a></td><td><a href="calibration/14.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_14.1576687</td><td align="right">76,687</td><td align="center">300.5</td><td align="center">3.6</td><td align="center">5.7%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_15.1656293.overview.png">coverage</a></td><td><a href="calibration/15.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_15.1656293</td><td align="right">56,293</td><td align="center">125.3</td><td align="center">2.1</td><td align="center">1.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_16.1751277.overview.png">coverage</a></td><td><a href="calibration/16.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_16.1751277</td><td align="right">51,277</td><td align="center">121.3</td><td align="center">2.1</td><td align="center">1.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_17.1850634.overview.png">coverage</a></td><td><a href="calibration/17.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_17.1850634</td><td align="right">50,634</td><td align="center">125.3</td><td align="center">1.8</td><td align="center">1.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_18.1944693.overview.png">coverage</a></td><td><a href="calibration/18.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_18.1944693</td><td align="right">44,693</td><td align="center">230.9</td><td align="center">3.3</td><td align="center">2.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_19.2041667.overview.png">coverage</a></td><td><a href="calibration/19.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_19.2041667</td><td align="right">41,667</td><td align="center">119.5</td><td align="center">2.2</td><td align="center">1.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_20.2134129.overview.png">coverage</a></td><td><a href="calibration/20.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_20.2134129</td><td align="right">34,129</td><td align="center">126.9</td><td align="center">2.0</td><td align="center">1.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_21.2231063.overview.png">coverage</a></td><td><a href="calibration/21.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_21.2231063</td><td align="right">31,063</td><td align="center">239.7</td><td align="center">3.5</td><td align="center">1.9%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_22.2327628.overview.png">coverage</a></td><td><a href="calibration/22.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_22.2327628</td><td align="right">27,628</td><td align="center">138.8</td><td align="center">2.0</td><td align="center">1.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_23.2427593.overview.png">coverage</a></td><td><a href="calibration/23.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_23.2427593</td><td align="right">27,593</td><td align="center">121.0</td><td align="center">2.0</td><td align="center">0.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_24.2525806.overview.png">coverage</a></td><td><a href="calibration/24.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_24.2525806</td><td align="right">25,806</td><td align="center">134.9</td><td align="center">1.8</td><td align="center">0.9%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_25.2623266.overview.png">coverage</a></td><td><a href="calibration/25.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_25.2623266</td><td align="right">23,266</td><td align="center">135.5</td><td align="center">2.1</td><td align="center">1.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_26.2720738.overview.png">coverage</a></td><td><a href="calibration/26.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_26.2720738</td><td align="right">20,738</td><td align="center">120.9</td><td align="center">1.9</td><td align="center">0.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_27.2820099.overview.png">coverage</a></td><td><a href="calibration/27.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_27.2820099</td><td align="right">20,099</td><td align="center">218.6</td><td align="center">3.1</td><td align="center">1.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_28.2919030.overview.png">coverage</a></td><td><a href="calibration/28.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_28.2919030</td><td align="right">19,030</td><td align="center">216.1</td><td align="center">2.9</td><td align="center">1.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_29.3015278.overview.png">coverage</a></td><td><a href="calibration/29.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_29.3015278</td><td align="right">15,278</td><td align="center">134.3</td><td align="center">2.2</td><td align="center">0.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_30.3110384.overview.png">coverage</a></td><td><a href="calibration/30.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_30.3110384</td><td align="right">10,384</td><td align="center">123.0</td><td align="center">1.4</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_31.328353.overview.png">coverage</a></td><td><a href="calibration/31.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_31.328353</td><td align="right">8,353</td><td align="center">141.0</td><td align="center">2.8</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_32.337638.overview.png">coverage</a></td><td><a href="calibration/32.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_32.337638</td><td align="right">7,638</td><td align="center">304.8</td><td align="center">3.8</td><td align="center">0.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_33.344101.overview.png">coverage</a></td><td><a href="calibration/33.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_33.344101</td><td align="right">4,101</td><td align="center">134.9</td><td align="center">2.2</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_34.353449.overview.png">coverage</a></td><td><a href="calibration/34.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_34.353449</td><td align="right">3,449</td><td align="center">448.9</td><td align="center">5.1</td><td align="center">0.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_35.361763.overview.png">coverage</a></td><td><a href="calibration/35.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_35.361763</td><td align="right">1,763</td><td align="center">259.6</td><td align="center">1.4</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/A37QDRAFT_scaffold_36.371564.overview.png">coverage</a></td><td><a href="calibration/36.unique_coverage.pdf">distribution</a></td><td>A37QDRAFT_scaffold_36.371564</td><td align="right">1,564</td><td align="center">427.1</td><td align="center">2.5</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>2,946,743</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit dispersion</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">12520</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;2</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">224</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.024</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>A37QDRAFT_scaffold_0.1282200</td><td align="center">0.80394</td></tr><tr >
<td>A37QDRAFT_scaffold_1.2281151</td><td align="center">0.80698</td></tr><tr >
<td>A37QDRAFT_scaffold_2.3252644</td><td align="center">0.80125</td></tr><tr >
<td>A37QDRAFT_scaffold_3.4207228</td><td align="center">0.80303</td></tr><tr >
<td>A37QDRAFT_scaffold_4.5202067</td><td align="center">0.80612</td></tr><tr >
<td>A37QDRAFT_scaffold_5.6181296</td><td align="center">0.80540</td></tr><tr >
<td>A37QDRAFT_scaffold_6.7175206</td><td align="center">0.80933</td></tr><tr >
<td>A37QDRAFT_scaffold_7.8155932</td><td align="center">0.79611</td></tr><tr >
<td>A37QDRAFT_scaffold_8.9138458</td><td align="center">0.81103</td></tr><tr >
<td>A37QDRAFT_scaffold_9.10119140</td><td align="center">0.80056</td></tr><tr >
<td>A37QDRAFT_scaffold_10.1198990</td><td align="center">0.80069</td></tr><tr >
<td>A37QDRAFT_scaffold_11.1287663</td><td align="center">0.80592</td></tr><tr >
<td>A37QDRAFT_scaffold_12.1382347</td><td align="center">0.80098</td></tr><tr >
<td>A37QDRAFT_scaffold_13.1479288</td><td align="center">0.81310</td></tr><tr >
<td>A37QDRAFT_scaffold_14.1576687</td><td align="center">0.68586</td></tr><tr >
<td>A37QDRAFT_scaffold_15.1656293</td><td align="center">0.80508</td></tr><tr >
<td>A37QDRAFT_scaffold_16.1751277</td><td align="center">0.80969</td></tr><tr >
<td>A37QDRAFT_scaffold_17.1850634</td><td align="center">0.80520</td></tr><tr >
<td>A37QDRAFT_scaffold_18.1944693</td><td align="center">0.72336</td></tr><tr >
<td>A37QDRAFT_scaffold_19.2041667</td><td align="center">0.81286</td></tr><tr >
<td>A37QDRAFT_scaffold_20.2134129</td><td align="center">0.80513</td></tr><tr >
<td>A37QDRAFT_scaffold_21.2231063</td><td align="center">0.71749</td></tr><tr >
<td>A37QDRAFT_scaffold_22.2327628</td><td align="center">0.79418</td></tr><tr >
<td>A37QDRAFT_scaffold_23.2427593</td><td align="center">0.81127</td></tr><tr >
<td>A37QDRAFT_scaffold_24.2525806</td><td align="center">0.79725</td></tr><tr >
<td>A37QDRAFT_scaffold_25.2623266</td><td align="center">0.77453</td></tr><tr >
<td>A37QDRAFT_scaffold_26.2720738</td><td align="center">0.80917</td></tr><tr >
<td>A37QDRAFT_scaffold_27.2820099</td><td align="center">0.73738</td></tr><tr >
<td>A37QDRAFT_scaffold_28.2919030</td><td align="center">0.73590</td></tr><tr >
<td>A37QDRAFT_scaffold_29.3015278</td><td align="center">0.80218</td></tr><tr >
<td>A37QDRAFT_scaffold_30.3110384</td><td align="center">0.80753</td></tr><tr >
<td>A37QDRAFT_scaffold_31.328353</td><td align="center">0.78982</td></tr><tr >
<td>A37QDRAFT_scaffold_32.337638</td><td align="center">0.69252</td></tr><tr >
<td>A37QDRAFT_scaffold_33.344101</td><td align="center">0.80127</td></tr><tr >
<td>A37QDRAFT_scaffold_34.353449</td><td align="center">0.61982</td></tr><tr >
<td>A37QDRAFT_scaffold_35.361763</td><td align="center">0.73738</td></tr><tr >
<td>A37QDRAFT_scaffold_36.371564</td><td align="center">0.67612</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.4.1</td></tr>
<tr><td>R</td><td>4.3.1</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>11:06:26&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:07:00&nbsp;24&nbsp;Jul&nbsp;2025</td><td>34&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>11:07:00&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:17:26&nbsp;24&nbsp;Jul&nbsp;2025</td><td>10&nbsp;minutes&nbsp;26&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>11:17:26&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:17:57&nbsp;24&nbsp;Jul&nbsp;2025</td><td>31&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>11:17:57&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:19:44&nbsp;24&nbsp;Jul&nbsp;2025</td><td>1&nbsp;minute&nbsp;47&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>11:19:44&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:19:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>4&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>11:19:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:22:19&nbsp;24&nbsp;Jul&nbsp;2025</td><td>2&nbsp;minutes&nbsp;31&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>11:22:19&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:23:06&nbsp;24&nbsp;Jul&nbsp;2025</td><td>47&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>11:23:06&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:24:20&nbsp;24&nbsp;Jul&nbsp;2025</td><td>1&nbsp;minute&nbsp;14&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>11:24:20&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:25:01&nbsp;24&nbsp;Jul&nbsp;2025</td><td>41&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>11:25:01&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:25:22&nbsp;24&nbsp;Jul&nbsp;2025</td><td>21&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>11:25:22&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:32:19&nbsp;24&nbsp;Jul&nbsp;2025</td><td>6&nbsp;minutes&nbsp;57&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>11:32:19&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:32:19&nbsp;24&nbsp;Jul&nbsp;2025</td><td>0&nbsp;seconds</td></tr>
<tr><td>Output</td><td>11:32:19&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:32:56&nbsp;24&nbsp;Jul&nbsp;2025</td><td>37&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>11:32:19&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:32:21&nbsp;24&nbsp;Jul&nbsp;2025</td><td>2&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>11:32:21&nbsp;24&nbsp;Jul&nbsp;2025</td><td>11:32:21&nbsp;24&nbsp;Jul&nbsp;2025</td><td>0&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>26&nbsp;minutes&nbsp;32&nbsp;seconds</b></td>
</tr>
</table></body></html>