<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Marginal Predictions</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.36.0&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<p>
<table border="0" cellspacing="1", cellpadding="3">
<tr><th colspan="12" align="left" class="read_alignment_header_row">Marginal read alignment evidence (highest frequency 20 of 24 shown, sorted by frequency from high to low)</th></tr>
<tr>
<th>&nbsp;</th>
<th>seq&nbsp;id</th>
<th colspan="2">position</th>
<th>ref</th>
<th>new</th>
<th>freq</th>
<th>score&nbsp;(cons/poly)</th>
<th>reads</th>
<th>annotation</th>
<th>genes</th>
<th width="100%">product</th>
</tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_104.html">*</a></td><td align="center">GCA_000011965.CP000032491611</td><td align="right">64,636</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">40.9%
</td><td align="center">18.8
/&nbsp;16.0
</td><td align="center">46</td><td align="center"><font class="snp_type_nonsynonymous">R330H</font>&nbsp;(C<font class="mutation_in_codon">G</font>C&rarr;C<font class="mutation_in_codon">A</font>C)&nbsp;</td><td align="center"><i>aruI</i></td><td align="left">putative 2&#8209;ketoarginine decarboxylase AruI</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_50.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">641,833</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">35.9%
</td><td align="center">61.4
/&nbsp;12.9
</td><td align="center">71</td><td align="center"><font class="snp_type_nonsynonymous">*353S</font>&nbsp;(T<font class="mutation_in_codon">A</font>G&rarr;T<font class="mutation_in_codon">C</font>G)&nbsp;</td><td align="center"><i>GMAPDCMF_00629</i></td><td align="left">IS110 family transposase ISSpo1</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_54.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">783,063</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">32.4%
</td><td align="center">38.7
/&nbsp;12.3
</td><td align="center">51</td><td align="center">intergenic&nbsp;(&#8209;44/+18)</td><td align="center"><i>GMAPDCMF_00753/GMAPDCMF_00754</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_80.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">2,826,871</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">32.0%
</td><td align="center">57.6
/&nbsp;10.8
</td><td align="center">63</td><td align="center"><font class="snp_type_nonsynonymous">P25S</font>&nbsp;(<font class="mutation_in_codon">C</font>CG&rarr;<font class="mutation_in_codon">T</font>CG)&nbsp;</td><td align="center"><i>mkl</i></td><td align="left">putative ribonucleotide transport ATP&#8209;binding protein mkl</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_76.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">2,300,084</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">31.6%
</td><td align="center">56.6
/&nbsp;12.1
</td><td align="center">48</td><td align="center"><font class="snp_type_nonsynonymous">A73G</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">G</font>G)&nbsp;</td><td align="center"><i>GMAPDCMF_02213</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_99.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">3,916,340</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">27.9%
</td><td align="center">221.1
/&nbsp;12.9
</td><td align="center">132</td><td align="center">intergenic&nbsp;(+149/&#8209;43)</td><td align="center"><i>aroK_2/GMAPDCMF_03789</i></td><td align="left">Shikimate kinase/Leu/Ile/Val&#8209;binding protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_53.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">673,674</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">27.5%
</td><td align="center">144.1
/&nbsp;19.4
</td><td align="center">80</td><td align="center"><font class="snp_type_nonsynonymous">Q124R</font>&nbsp;(C<font class="mutation_in_codon">A</font>G&rarr;C<font class="mutation_in_codon">G</font>G)&nbsp;</td><td align="center"><i>GMAPDCMF_00656</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_83.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">3,180,675</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">27.5%
</td><td align="center">137.7
/&nbsp;19.1
</td><td align="center">91</td><td align="center"><font class="snp_type_synonymous">S125S</font>&nbsp;(TC<font class="mutation_in_codon">A</font>&rarr;TC<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>tadA</i></td><td align="left">tRNA&#8209;specific adenosine deaminase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_70.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">1,945,729</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">27.4%
</td><td align="center">127.8
/&nbsp;19.3
</td><td align="center">68</td><td align="center"><font class="snp_type_synonymous">A207A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>hexR</i></td><td align="left">HTH&#8209;type transcriptional regulator HexR</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_92.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">3,403,819</td><td align="right">0</td><td align="center">G</td><td align="center">A</td><td align="right">27.0%
</td><td align="center">165.4
/&nbsp;13.0
</td><td align="center">89</td><td align="center"><font class="snp_type_nonsynonymous">L43F</font>&nbsp;(<font class="mutation_in_codon">C</font>TT&rarr;<font class="mutation_in_codon">T</font>TT)&nbsp;</td><td align="center"><i>gucD</i></td><td align="left">Alpha&#8209;ketoglutaric semialdehyde dehydrogenase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_94.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">3,492,796</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">25.7%
</td><td align="center">122.3
/&nbsp;16.4
</td><td align="center">110</td><td align="center">intergenic&nbsp;(&#8209;45/+34)</td><td align="center"><i>GMAPDCMF_03367/livH_7</i></td><td align="left">hypothetical protein/High&#8209;affinity branched&#8209;chain amino acid transport system permease protein LivH</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_109.html">*</a></td><td align="center">GCA_000011965.CP000032491611</td><td align="right">248,257</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">25.2%
</td><td align="center">252.7
/&nbsp;12.6
</td><td align="center">132</td><td align="center"><font class="snp_type_nonsense">S48*</font>&nbsp;(T<font class="mutation_in_codon">C</font>G&rarr;T<font class="mutation_in_codon">A</font>G)&nbsp;</td><td align="center"><i>mftC</i></td><td align="left">Putative mycofactocin radical SAM maturase MftC</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_100.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">3,919,431</td><td align="right">0</td><td align="center">G</td><td align="center">T</td><td align="right">24.0%
</td><td align="center">163.8
/&nbsp;24.0
</td><td align="center">75</td><td align="center"><font class="snp_type_nonsynonymous">G88V</font>&nbsp;(G<font class="mutation_in_codon">G</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td><td align="center"><i>livH_8</i></td><td align="left">High&#8209;affinity branched&#8209;chain amino acid transport system permease protein LivH</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_79.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">2,761,747</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">23.6%
</td><td align="center">174.8
/&nbsp;20.8
</td><td align="center">82</td><td align="center">intergenic&nbsp;(+14/&#8209;84)</td><td align="center"><i>GMAPDCMF_02670/ipuC</i></td><td align="left">Monocarboxylate 2&#8209;oxoacid&#8209;binding periplasmic protein/Glutamate&#8209;&#8209;isopropylamine ligase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_51.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">641,844</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">22.6%
</td><td align="center">133.6
/&nbsp;10.7
</td><td align="center">84</td><td align="center"><font class="snp_type_nonsynonymous">L349F</font>&nbsp;(TT<font class="mutation_in_codon">G</font>&rarr;TT<font class="mutation_in_codon">T</font>)&nbsp;</td><td align="center"><i>GMAPDCMF_00629</i></td><td align="left">IS110 family transposase ISSpo1</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_52.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">669,530</td><td align="right">0</td><td align="center">G</td><td align="center">A</td><td align="right">22.3%
</td><td align="center">275.1
/&nbsp;13.7
</td><td align="center">121</td><td align="center"><font class="snp_type_nonsynonymous">L298F</font>&nbsp;(<font class="mutation_in_codon">C</font>TT&rarr;<font class="mutation_in_codon">T</font>TT)&nbsp;</td><td align="center"><i>GMAPDCMF_00653</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_105.html">*</a></td><td align="center">GCA_000011965.CP000032491611</td><td align="right">108,318</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">21.9%
</td><td align="center">246.5
/&nbsp;14.2
</td><td align="center">123</td><td align="center">intergenic&nbsp;(&#8209;77/&#8209;17)</td><td align="center"><i>gcvA_13/GMAPDCMF_04098</i></td><td align="left">Glycine cleavage system transcriptional activator/L&#8209;lactate transporter</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_77.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">2,504,743</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">21.7%
</td><td align="center">169.8
/&nbsp;16.2
</td><td align="center">79</td><td align="center"><font class="snp_type_nonsynonymous">A309G</font>&nbsp;(G<font class="mutation_in_codon">C</font>G&rarr;G<font class="mutation_in_codon">G</font>G)&nbsp;</td><td align="center"><i>rcsC_6</i></td><td align="left">Sensor histidine kinase RcsC</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_44.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">122,298</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">21.6%
</td><td align="center">319.8
/&nbsp;13.5
</td><td align="center">153</td><td align="center"><font class="snp_type_nonsense">S390*</font>&nbsp;(T<font class="mutation_in_codon">C</font>G&rarr;T<font class="mutation_in_codon">A</font>G)&nbsp;</td><td align="center"><i>GMAPDCMF_00117</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_87.html">*</a></td><td align="center">GCA_000011965.CP0000314109437</td><td align="right">3,236,455</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">20.6%
</td><td align="center">163.4
/&nbsp;14.4
</td><td align="center">73</td><td align="center">intergenic&nbsp;(+65/+13)</td><td align="center"><i>msbA/eda_1</i></td><td align="left">Lipid A export ATP&#8209;binding/permease protein MsbA/KHG/KDPG aldolase</td></tr>
</table>

<p>
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="12" align="left" class="new_junction_header_row">Marginal new junction evidence (lowest skew 10 of 84 shown)</th></tr>
<!-- Header Lines for New Junction -->
<tr>
<th colspan="2">&nbsp;</th>
<th>seq&nbsp;id</th>
<th>position</th>
<th>reads&nbsp;(cov)</th>
<th>reads&nbsp;(cov)</th>
<th>score</th>
<th>skew</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">product</th>
</tr>

<!-- Item Lines for New Junction -->
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_183.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_183.html">?</a></td>
<td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">=&nbsp;3666809</td><td align="center" class="junction_gene">175 (1.450)</td><td rowspan="2" align="center">29 (0.260)</td>
<td rowspan="2" align="center">18/292</td>
<td rowspan="2" align="center">3.8</td>
<td rowspan="2" align="center">14.9%</td>
<td align="center" class="junction_gene">coding&nbsp;(36/669&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_03541</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_183.html">?</a></td><td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">=&nbsp;3666797</td>
<td align="center" class="junction_gene">170 (1.530)</td><td align="center" class="junction_gene">coding&nbsp;(24/669&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_03541</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="2"><a href="evidence/JC_151.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_151.html">?</a></td>
<td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">=&nbsp;1929310</td><td align="center" class="junction_gene">92 (0.760)</td><td rowspan="2" align="center">28 (0.260)</td>
<td rowspan="2" align="center">16/284</td>
<td rowspan="2" align="center">4.2</td>
<td rowspan="2" align="center">25.1%</td>
<td align="center" class="junction_gene">coding&nbsp;(257/285&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_01860</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="1"><a href="evidence/JC_SIDE_2_151.html">?</a></td><td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">=&nbsp;1929294</td>
<td align="center" class="junction_gene">84 (0.780)</td><td align="center" class="junction_gene">coding&nbsp;(273/285&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_01860</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_114.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_114.html">?</a></td>
<td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">1893&nbsp;=</td><td align="center" class="junction_gene">172 (1.430)</td><td rowspan="2" align="center">11 (0.100)</td>
<td rowspan="2" align="center">10/276</td>
<td rowspan="2" align="center">5.8</td>
<td rowspan="2" align="center">7.0%</td>
<td align="center" class="junction_gene">coding&nbsp;(28/615&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>rsmG</i></td>
<td class="junction_gene">Ribosomal RNA small subunit methyltransferase G</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_114.html">?</a></td><td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">1913&nbsp;=</td>
<td align="center" class="junction_gene">143 (1.360)</td><td align="center" class="junction_gene">coding&nbsp;(48/615&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>rsmG</i></td>
<td class="junction_gene">Ribosomal RNA small subunit methyltransferase G</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="2"><a href="evidence/JC_194.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_194.html">?</a></td>
<td rowspan="1" class="junction_gene">GCA_000011965.CP000032491611</td>
<td align="center" class="junction_gene">106981&nbsp;=</td><td align="center" class="junction_gene">141 (1.050)</td><td rowspan="2" align="center">23 (0.180)</td>
<td rowspan="2" align="center">11/302</td>
<td rowspan="2" align="center">5.9</td>
<td rowspan="2" align="center">15.0%</td>
<td align="center" class="junction_gene">intergenic&nbsp;(+337/+367)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_04096/gcvA_13</i></td>
<td class="junction_gene">hypothetical protein/Glycine cleavage system transcriptional activator</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="1"><a href="evidence/JC_SIDE_2_194.html">?</a></td><td rowspan="1" class="junction_gene">GCA_000011965.CP000032491611</td>
<td align="center" class="junction_gene">107002&nbsp;=</td>
<td align="center" class="junction_gene">126 (0.990)</td><td align="center" class="junction_gene">intergenic&nbsp;(+358/+346)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_04096/gcvA_13</i></td>
<td class="junction_gene">hypothetical protein/Glycine cleavage system transcriptional activator</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_144.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_144.html">?</a></td>
<td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">1632632&nbsp;=</td><td align="center" class="junction_gene">136 (1.130)</td><td rowspan="2" align="center">21 (0.180)</td>
<td rowspan="2" align="center">11/300</td>
<td rowspan="2" align="center">5.9</td>
<td rowspan="2" align="center">13.6%</td>
<td align="center" class="junction_gene">coding&nbsp;(189/942&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_01583</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_144.html">?</a></td><td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">1632661&nbsp;=</td>
<td align="center" class="junction_gene">137 (1.200)</td><td align="center" class="junction_gene">coding&nbsp;(218/942&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_01583</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="2"><a href="evidence/JC_135.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_135.html">?</a></td>
<td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">=&nbsp;741976</td><td align="center" class="junction_gene">138 (1.150)</td><td rowspan="2" align="center">15 (0.130)</td>
<td rowspan="2" align="center">11/298</td>
<td rowspan="2" align="center">5.9</td>
<td rowspan="2" align="center">10.7%</td>
<td align="center" class="junction_gene">coding&nbsp;(201/1710&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>sasA_3</i></td>
<td class="junction_gene">Adaptive&#8209;response sensory&#8209;kinase SasA</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="1"><a href="evidence/JC_SIDE_2_135.html">?</a></td><td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">=&nbsp;741994</td>
<td align="center" class="junction_gene">120 (1.060)</td><td align="center" class="junction_gene">coding&nbsp;(219/1710&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>sasA_3</i></td>
<td class="junction_gene">Adaptive&#8209;response sensory&#8209;kinase SasA</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_138.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_138.html">?</a></td>
<td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">=&nbsp;1121549</td><td align="center" class="junction_gene">114 (0.950)</td><td rowspan="2" align="center">19 (0.170)</td>
<td rowspan="2" align="center">10/292</td>
<td rowspan="2" align="center">6.1</td>
<td rowspan="2" align="center">15.2%</td>
<td align="center" class="junction_gene">coding&nbsp;(875/903&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_01087</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_138.html">?</a></td><td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">=&nbsp;1121537</td>
<td align="center" class="junction_gene">106 (0.950)</td><td align="center" class="junction_gene">coding&nbsp;(887/903&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_01087</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="2"><a href="evidence/JC_126.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_126.html">?</a></td>
<td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">501605&nbsp;=</td><td align="center" class="junction_gene">186 (1.550)</td><td rowspan="2" align="center">14 (0.130)</td>
<td rowspan="2" align="center">10/288</td>
<td rowspan="2" align="center">6.1</td>
<td rowspan="2" align="center">7.5%</td>
<td align="center" class="junction_gene">coding&nbsp;(516/624&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_00509</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="1"><a href="evidence/JC_SIDE_2_126.html">?</a></td><td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">501619&nbsp;=</td>
<td align="center" class="junction_gene">177 (1.610)</td><td align="center" class="junction_gene">coding&nbsp;(530/624&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_00509</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_167.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_167.html">?</a></td>
<td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">2706578&nbsp;=</td><td align="center" class="junction_gene">147 (1.220)</td><td rowspan="2" align="center">9 (0.080)</td>
<td rowspan="2" align="center">9/284</td>
<td rowspan="2" align="center">6.4</td>
<td rowspan="2" align="center">6.8%</td>
<td align="center" class="junction_gene">intergenic&nbsp;(&#8209;36/&#8209;66)</td>
<td align="center" class="junction_gene"><i>hbpA/GMAPDCMF_02620</i></td>
<td class="junction_gene">Heme&#8209;binding protein A/hypothetical protein</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_167.html">?</a></td><td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">2706617&nbsp;=</td>
<td align="center" class="junction_gene">116 (1.070)</td><td align="center" class="junction_gene">intergenic&nbsp;(&#8209;75/&#8209;27)</td>
<td align="center" class="junction_gene"><i>hbpA/GMAPDCMF_02620</i></td>
<td class="junction_gene">Heme&#8209;binding protein A/hypothetical protein</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="2"><a href="evidence/JC_190.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_190.html">?</a></td>
<td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">=&nbsp;4029511</td><td align="center" class="junction_gene">129 (1.070)</td><td rowspan="2" align="center">13 (0.110)</td>
<td rowspan="2" align="center">9/298</td>
<td rowspan="2" align="center">6.6</td>
<td rowspan="2" align="center">9.7%</td>
<td align="center" class="junction_gene">coding&nbsp;(51/3117&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_03899</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="1"><a href="evidence/JC_SIDE_2_190.html">?</a></td><td rowspan="1" class="junction_gene">GCA_000011965.CP0000314109437</td>
<td align="center" class="junction_gene">=&nbsp;4029517</td>
<td align="center" class="junction_gene">121 (1.070)</td><td align="center" class="junction_gene">coding&nbsp;(57/3117&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>GMAPDCMF_03899</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
</table>
</body></html>