<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.36.0&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/PNBC3Y_2_S23_R1_001.error_rates.pdf">errors</a></td><td>PNBC3Y_2_S23_R1_001</td><td align="right">7,562,729</td><td align="right">1,202,473,911</td><td align="right">100.0%</td><td align="right">159.0&nbsp;bases</td><td align="right">159&nbsp;bases</td><td align="right">88.9%</td></tr><tr ><td><a href="calibration/PNBC3Y_2_S23_R2_001.error_rates.pdf">errors</a></td><td>PNBC3Y_2_S23_R2_001</td><td align="right">7,549,956</td><td align="right">1,200,443,004</td><td align="right">99.8%</td><td align="right">159.0&nbsp;bases</td><td align="right">159&nbsp;bases</td><td align="right">88.5%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>15,112,685</b></td><td align="right"><b>2,402,916,915</b></td><td align="right">99.9%</td><td align="right">159.0&nbsp;bases</td><td align="right"><b>159</b>&nbsp;bases</td><td align="right">88.7%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit dispersion</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/NZ_AXWP01000001.1.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000001.1</td><td align="right">483,335</td><td align="center">375.2</td><td align="center">4.9</td><td align="center">8.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519477.1.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>NZ_KI519477.1</td><td align="right">425,584</td><td align="center">478.6</td><td align="center">6.9</td><td align="center">9.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000008.1.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000008.1</td><td align="right">382,303</td><td align="center">483.0</td><td align="center">4.7</td><td align="center">8.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519478.1.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>NZ_KI519478.1</td><td align="right">356,521</td><td align="center">543.5</td><td align="center">10.1</td><td align="center">9.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519479.1.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>NZ_KI519479.1</td><td align="right">253,925</td><td align="center">387.1</td><td align="center">4.0</td><td align="center">4.7%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519480.1.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>NZ_KI519480.1</td><td align="right">189,234</td><td align="center">415.7</td><td align="center">4.0</td><td align="center">3.7%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519481.1.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>NZ_KI519481.1</td><td align="right">158,800</td><td align="center">385.9</td><td align="center">3.8</td><td align="center">2.9%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519482.1.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>NZ_KI519482.1</td><td align="right">154,526</td><td align="center">389.7</td><td align="center">3.6</td><td align="center">2.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000025.1.overview.png">coverage</a></td><td><a href="calibration/8.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000025.1</td><td align="right">120,022</td><td align="center">371.5</td><td align="center">3.8</td><td align="center">2.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000026.1.overview.png">coverage</a></td><td><a href="calibration/9.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000026.1</td><td align="right">105,827</td><td align="center">363.9</td><td align="center">3.3</td><td align="center">1.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000027.1.overview.png">coverage</a></td><td><a href="calibration/10.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000027.1</td><td align="right">103,287</td><td align="center">530.7</td><td align="center">5.3</td><td align="center">2.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519483.1.overview.png">coverage</a></td><td><a href="calibration/11.unique_coverage.pdf">distribution</a></td><td>NZ_KI519483.1</td><td align="right">89,264</td><td align="center">341.2</td><td align="center">3.7</td><td align="center">1.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000030.1.overview.png">coverage</a></td><td><a href="calibration/12.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000030.1</td><td align="right">87,830</td><td align="center">523.0</td><td align="center">5.2</td><td align="center">2.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519484.1.overview.png">coverage</a></td><td><a href="calibration/13.unique_coverage.pdf">distribution</a></td><td>NZ_KI519484.1</td><td align="right">86,193</td><td align="center">346.3</td><td align="center">3.8</td><td align="center">1.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519485.1.overview.png">coverage</a></td><td><a href="calibration/14.unique_coverage.pdf">distribution</a></td><td>NZ_KI519485.1</td><td align="right">85,175</td><td align="center">487.9</td><td align="center">4.8</td><td align="center">1.9%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000036.1.overview.png">coverage</a></td><td><a href="calibration/15.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000036.1</td><td align="right">80,378</td><td align="center">346.9</td><td align="center">3.2</td><td align="center">1.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000037.1.overview.png">coverage</a></td><td><a href="calibration/16.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000037.1</td><td align="right">80,071</td><td align="center">480.7</td><td align="center">4.5</td><td align="center">1.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000038.1.overview.png">coverage</a></td><td><a href="calibration/17.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000038.1</td><td align="right">78,045</td><td align="center">523.4</td><td align="center">4.8</td><td align="center">1.9%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519486.1.overview.png">coverage</a></td><td><a href="calibration/18.unique_coverage.pdf">distribution</a></td><td>NZ_KI519486.1</td><td align="right">78,782</td><td align="center">394.7</td><td align="center">4.4</td><td align="center">1.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000041.1.overview.png">coverage</a></td><td><a href="calibration/19.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000041.1</td><td align="right">72,912</td><td align="center">451.5</td><td align="center">6.7</td><td align="center">1.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519487.1.overview.png">coverage</a></td><td><a href="calibration/20.unique_coverage.pdf">distribution</a></td><td>NZ_KI519487.1</td><td align="right">71,535</td><td align="center">445.5</td><td align="center">4.6</td><td align="center">1.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000045.1.overview.png">coverage</a></td><td><a href="calibration/21.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000045.1</td><td align="right">71,670</td><td align="center">546.8</td><td align="center">5.8</td><td align="center">1.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519488.1.overview.png">coverage</a></td><td><a href="calibration/22.unique_coverage.pdf">distribution</a></td><td>NZ_KI519488.1</td><td align="right">59,723</td><td align="center">363.8</td><td align="center">3.9</td><td align="center">1.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000048.1.overview.png">coverage</a></td><td><a href="calibration/23.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000048.1</td><td align="right">59,130</td><td align="center">496.3</td><td align="center">4.6</td><td align="center">1.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000049.1.overview.png">coverage</a></td><td><a href="calibration/24.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000049.1</td><td align="right">57,016</td><td align="center">421.3</td><td align="center">4.0</td><td align="center">1.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000050.1.overview.png">coverage</a></td><td><a href="calibration/25.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000050.1</td><td align="right">56,824</td><td align="center">499.4</td><td align="center">4.1</td><td align="center">1.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000051.1.overview.png">coverage</a></td><td><a href="calibration/26.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000051.1</td><td align="right">52,777</td><td align="center">430.5</td><td align="center">5.5</td><td align="center">1.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000052.1.overview.png">coverage</a></td><td><a href="calibration/27.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000052.1</td><td align="right">51,318</td><td align="center">438.2</td><td align="center">3.7</td><td align="center">1.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519489.1.overview.png">coverage</a></td><td><a href="calibration/28.unique_coverage.pdf">distribution</a></td><td>NZ_KI519489.1</td><td align="right">47,148</td><td align="center">451.6</td><td align="center">9.2</td><td align="center">1.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000055.1.overview.png">coverage</a></td><td><a href="calibration/29.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000055.1</td><td align="right">43,333</td><td align="center">479.7</td><td align="center">5.2</td><td align="center">1.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000056.1.overview.png">coverage</a></td><td><a href="calibration/30.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000056.1</td><td align="right">40,004</td><td align="center">340.8</td><td align="center">3.4</td><td align="center">0.7%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000057.1.overview.png">coverage</a></td><td><a href="calibration/31.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000057.1</td><td align="right">37,599</td><td align="center">504.8</td><td align="center">6.1</td><td align="center">0.9%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000058.1.overview.png">coverage</a></td><td><a href="calibration/32.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000058.1</td><td align="right">37,036</td><td align="center">428.9</td><td align="center">4.5</td><td align="center">0.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000059.1.overview.png">coverage</a></td><td><a href="calibration/33.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000059.1</td><td align="right">36,634</td><td align="center">467.6</td><td align="center">4.6</td><td align="center">0.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519490.1.overview.png">coverage</a></td><td><a href="calibration/34.unique_coverage.pdf">distribution</a></td><td>NZ_KI519490.1</td><td align="right">33,421</td><td align="center">486.5</td><td align="center">4.8</td><td align="center">0.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519491.1.overview.png">coverage</a></td><td><a href="calibration/35.unique_coverage.pdf">distribution</a></td><td>NZ_KI519491.1</td><td align="right">33,532</td><td align="center">476.9</td><td align="center">4.8</td><td align="center">0.7%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000064.1.overview.png">coverage</a></td><td><a href="calibration/36.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000064.1</td><td align="right">32,130</td><td align="center">525.9</td><td align="center">4.6</td><td align="center">0.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000065.1.overview.png">coverage</a></td><td><a href="calibration/37.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000065.1</td><td align="right">31,524</td><td align="center">496.2</td><td align="center">3.8</td><td align="center">0.7%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000066.1.overview.png">coverage</a></td><td><a href="calibration/38.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000066.1</td><td align="right">31,275</td><td align="center">398.5</td><td align="center">4.1</td><td align="center">0.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000067.1.overview.png">coverage</a></td><td><a href="calibration/39.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000067.1</td><td align="right">26,798</td><td align="center">492.2</td><td align="center">4.2</td><td align="center">0.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000068.1.overview.png">coverage</a></td><td><a href="calibration/40.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000068.1</td><td align="right">24,680</td><td align="center">409.1</td><td align="center">3.2</td><td align="center">0.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000069.1.overview.png">coverage</a></td><td><a href="calibration/41.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000069.1</td><td align="right">23,780</td><td align="center">434.7</td><td align="center">3.5</td><td align="center">0.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000070.1.overview.png">coverage</a></td><td><a href="calibration/42.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000070.1</td><td align="right">21,593</td><td align="center">497.3</td><td align="center">3.6</td><td align="center">0.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000071.1.overview.png">coverage</a></td><td><a href="calibration/43.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000071.1</td><td align="right">21,563</td><td align="center">520.6</td><td align="center">3.5</td><td align="center">0.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519492.1.overview.png">coverage</a></td><td><a href="calibration/44.unique_coverage.pdf">distribution</a></td><td>NZ_KI519492.1</td><td align="right">21,286</td><td align="center">426.8</td><td align="center">3.7</td><td align="center">0.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000074.1.overview.png">coverage</a></td><td><a href="calibration/45.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000074.1</td><td align="right">19,792</td><td align="center">367.4</td><td align="center">3.8</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000075.1.overview.png">coverage</a></td><td><a href="calibration/46.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000075.1</td><td align="right">15,898</td><td align="center">509.8</td><td align="center">5.3</td><td align="center">0.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_KI519493.1.overview.png">coverage</a></td><td><a href="calibration/47.unique_coverage.pdf">distribution</a></td><td>NZ_KI519493.1</td><td align="right">16,227</td><td align="center">517.0</td><td align="center">4.5</td><td align="center">0.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000078.1.overview.png">coverage</a></td><td><a href="calibration/48.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000078.1</td><td align="right">15,701</td><td align="center">445.1</td><td align="center">4.9</td><td align="center">0.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000079.1.overview.png">coverage</a></td><td><a href="calibration/49.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000079.1</td><td align="right">14,417</td><td align="center">456.8</td><td align="center">3.5</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000080.1.overview.png">coverage</a></td><td><a href="calibration/50.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000080.1</td><td align="right">14,114</td><td align="center">518.5</td><td align="center">4.2</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000081.1.overview.png">coverage</a></td><td><a href="calibration/51.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000081.1</td><td align="right">13,316</td><td align="center">443.4</td><td align="center">3.4</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000082.1.overview.png">coverage</a></td><td><a href="calibration/52.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000082.1</td><td align="right">12,863</td><td align="center">511.1</td><td align="center">3.4</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000083.1.overview.png">coverage</a></td><td><a href="calibration/53.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000083.1</td><td align="right">12,841</td><td align="center">429.2</td><td align="center">4.1</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000084.1.overview.png">coverage</a></td><td><a href="calibration/54.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000084.1</td><td align="right">12,083</td><td align="center">502.7</td><td align="center">3.7</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000085.1.overview.png">coverage</a></td><td><a href="calibration/55.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000085.1</td><td align="right">8,976</td><td align="center">400.7</td><td align="center">2.3</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000086.1.overview.png">coverage</a></td><td><a href="calibration/56.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000086.1</td><td align="right">8,568</td><td align="center">463.2</td><td align="center">4.5</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000087.1.overview.png">coverage</a></td><td><a href="calibration/57.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000087.1</td><td align="right">7,526</td><td align="center">552.5</td><td align="center">4.6</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000088.1.overview.png">coverage</a></td><td><a href="calibration/58.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000088.1</td><td align="right">6,520</td><td align="center">244.4</td><td align="center">2.6</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000089.1.overview.png">coverage</a></td><td><a href="calibration/59.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000089.1</td><td align="right">6,402</td><td align="center">446.8</td><td align="center">3.5</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000090.1.overview.png">coverage</a></td><td><a href="calibration/60.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000090.1</td><td align="right">5,385</td><td align="center">375.1</td><td align="center">3.1</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000091.1.overview.png">coverage</a></td><td><a href="calibration/61.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000091.1</td><td align="right">5,269</td><td align="center">562.0</td><td align="center">6.9</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000092.1.overview.png">coverage</a></td><td><a href="calibration/62.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000092.1</td><td align="right">5,745</td><td align="center">437.4</td><td align="center">7.7</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000093.1.overview.png">coverage</a></td><td><a href="calibration/63.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000093.1</td><td align="right">5,118</td><td align="center">454.0</td><td align="center">4.3</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000094.1.overview.png">coverage</a></td><td><a href="calibration/64.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000094.1</td><td align="right">4,959</td><td align="center">492.0</td><td align="center">2.4</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000095.1.overview.png">coverage</a></td><td><a href="calibration/65.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000095.1</td><td align="right">4,806</td><td align="center">424.8</td><td align="center">3.7</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000096.1.overview.png">coverage</a></td><td><a href="calibration/66.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000096.1</td><td align="right">4,515</td><td align="center">434.4</td><td align="center">3.4</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr class="gray_table_row"><td><a href="evidence/NZ_AXWP01000097.1.overview.png">coverage</a></td><td><a href="calibration/67.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000097.1</td><td align="right">3,747</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000098.1.overview.png">coverage</a></td><td><a href="calibration/68.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000098.1</td><td align="right">3,439</td><td align="center">490.6</td><td align="center">3.6</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000099.1.overview.png">coverage</a></td><td><a href="calibration/69.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000099.1</td><td align="right">2,724</td><td align="center">386.6</td><td align="center">3.3</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000100.1.overview.png">coverage</a></td><td><a href="calibration/70.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000100.1</td><td align="right">2,618</td><td align="center">499.7</td><td align="center">2.1</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000101.1.overview.png">coverage</a></td><td><a href="calibration/71.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000101.1</td><td align="right">2,280</td><td align="center">438.9</td><td align="center">2.9</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000102.1.overview.png">coverage</a></td><td><a href="calibration/72.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000102.1</td><td align="right">2,226</td><td align="center">457.0</td><td align="center">2.8</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000103.1.overview.png">coverage</a></td><td><a href="calibration/73.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000103.1</td><td align="right">2,126</td><td align="center">424.8</td><td align="center">4.3</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000104.1.overview.png">coverage</a></td><td><a href="calibration/74.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000104.1</td><td align="right">1,864</td><td align="center">424.0</td><td align="center">5.4</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000105.1.overview.png">coverage</a></td><td><a href="calibration/75.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000105.1</td><td align="right">1,655</td><td align="center">3400.1</td><td align="center">21.8</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr class="gray_table_row"><td><a href="evidence/NZ_AXWP01000106.1.overview.png">coverage</a></td><td><a href="calibration/76.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000106.1</td><td align="right">1,654</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr class="gray_table_row"><td><a href="evidence/NZ_AXWP01000107.1.overview.png">coverage</a></td><td><a href="calibration/77.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000107.1</td><td align="right">1,653</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr class="gray_table_row"><td><a href="evidence/NZ_AXWP01000108.1.overview.png">coverage</a></td><td><a href="calibration/78.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000108.1</td><td align="right">1,438</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000109.1.overview.png">coverage</a></td><td><a href="calibration/79.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000109.1</td><td align="right">1,061</td><td align="center">3632.9</td><td align="center">11.5</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/NZ_AXWP01000110.1.overview.png">coverage</a></td><td><a href="calibration/80.unique_coverage.pdf">distribution</a></td><td>NZ_AXWP01000110.1</td><td align="right">1,028</td><td align="center">420.5</td><td align="center">4.2</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>4,745,897</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit dispersion</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<p><span class="gray_table_row">Insufficient coverage</span> Reference sequence counted as entirely deleted due to low coverage. Try either the <code>-t,--targeted-sequencing</code> or the <code>-c,--contig-reference </code> option if you want mutations called for these reference sequences.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">100000</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">753</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.053</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>NZ_AXWP01000001.1</td><td align="center">0.63943</td></tr><tr >
<td>NZ_KI519477.1</td><td align="center">0.60484</td></tr><tr >
<td>NZ_AXWP01000008.1</td><td align="center">0.59854</td></tr><tr >
<td>NZ_KI519478.1</td><td align="center">0.58173</td></tr><tr >
<td>NZ_KI519479.1</td><td align="center">0.64042</td></tr><tr >
<td>NZ_KI519480.1</td><td align="center">0.62606</td></tr><tr >
<td>NZ_KI519481.1</td><td align="center">0.63705</td></tr><tr >
<td>NZ_KI519482.1</td><td align="center">0.64041</td></tr><tr >
<td>NZ_AXWP01000025.1</td><td align="center">0.64425</td></tr><tr >
<td>NZ_AXWP01000026.1</td><td align="center">0.64533</td></tr><tr >
<td>NZ_AXWP01000027.1</td><td align="center">0.58188</td></tr><tr >
<td>NZ_KI519483.1</td><td align="center">0.65621</td></tr><tr >
<td>NZ_AXWP01000030.1</td><td align="center">0.58817</td></tr><tr >
<td>NZ_KI519484.1</td><td align="center">0.65583</td></tr><tr >
<td>NZ_KI519485.1</td><td align="center">0.60134</td></tr><tr >
<td>NZ_AXWP01000036.1</td><td align="center">0.65170</td></tr><tr >
<td>NZ_AXWP01000037.1</td><td align="center">0.60076</td></tr><tr >
<td>NZ_AXWP01000038.1</td><td align="center">0.58670</td></tr><tr >
<td>NZ_KI519486.1</td><td align="center">0.64143</td></tr><tr >
<td>NZ_AXWP01000041.1</td><td align="center">0.60527</td></tr><tr >
<td>NZ_KI519487.1</td><td align="center">0.63507</td></tr><tr >
<td>NZ_AXWP01000045.1</td><td align="center">0.58864</td></tr><tr >
<td>NZ_KI519488.1</td><td align="center">0.65118</td></tr><tr >
<td>NZ_AXWP01000048.1</td><td align="center">0.59807</td></tr><tr >
<td>NZ_AXWP01000049.1</td><td align="center">0.62327</td></tr><tr >
<td>NZ_AXWP01000050.1</td><td align="center">0.59232</td></tr><tr >
<td>NZ_AXWP01000051.1</td><td align="center">0.62432</td></tr><tr >
<td>NZ_AXWP01000052.1</td><td align="center">0.61474</td></tr><tr >
<td>NZ_KI519489.1</td><td align="center">0.61671</td></tr><tr >
<td>NZ_AXWP01000055.1</td><td align="center">0.59767</td></tr><tr >
<td>NZ_AXWP01000056.1</td><td align="center">0.65475</td></tr><tr >
<td>NZ_AXWP01000057.1</td><td align="center">0.59354</td></tr><tr >
<td>NZ_AXWP01000058.1</td><td align="center">0.61890</td></tr><tr >
<td>NZ_AXWP01000059.1</td><td align="center">0.60067</td></tr><tr >
<td>NZ_KI519490.1</td><td align="center">0.60256</td></tr><tr >
<td>NZ_KI519491.1</td><td align="center">0.61007</td></tr><tr >
<td>NZ_AXWP01000064.1</td><td align="center">0.58909</td></tr><tr >
<td>NZ_AXWP01000065.1</td><td align="center">0.59471</td></tr><tr >
<td>NZ_AXWP01000066.1</td><td align="center">0.63702</td></tr><tr >
<td>NZ_AXWP01000067.1</td><td align="center">0.59742</td></tr><tr >
<td>NZ_AXWP01000068.1</td><td align="center">0.62972</td></tr><tr >
<td>NZ_AXWP01000069.1</td><td align="center">0.61465</td></tr><tr >
<td>NZ_AXWP01000070.1</td><td align="center">0.59315</td></tr><tr >
<td>NZ_AXWP01000071.1</td><td align="center">0.59161</td></tr><tr >
<td>NZ_KI519492.1</td><td align="center">0.63181</td></tr><tr >
<td>NZ_AXWP01000074.1</td><td align="center">0.64204</td></tr><tr >
<td>NZ_AXWP01000075.1</td><td align="center">0.59324</td></tr><tr >
<td>NZ_KI519493.1</td><td align="center">0.60742</td></tr><tr >
<td>NZ_AXWP01000078.1</td><td align="center">0.61092</td></tr><tr >
<td>NZ_AXWP01000079.1</td><td align="center">0.61006</td></tr><tr >
<td>NZ_AXWP01000080.1</td><td align="center">0.58717</td></tr><tr >
<td>NZ_AXWP01000081.1</td><td align="center">0.61332</td></tr><tr >
<td>NZ_AXWP01000082.1</td><td align="center">0.58512</td></tr><tr >
<td>NZ_AXWP01000083.1</td><td align="center">0.62254</td></tr><tr >
<td>NZ_AXWP01000084.1</td><td align="center">0.60254</td></tr><tr >
<td>NZ_AXWP01000085.1</td><td align="center">0.62112</td></tr><tr >
<td>NZ_AXWP01000086.1</td><td align="center">0.61760</td></tr><tr >
<td>NZ_AXWP01000087.1</td><td align="center">0.58469</td></tr><tr >
<td>NZ_AXWP01000088.1</td><td align="center">0.70891</td></tr><tr >
<td>NZ_AXWP01000089.1</td><td align="center">0.60337</td></tr><tr >
<td>NZ_AXWP01000090.1</td><td align="center">0.63851</td></tr><tr >
<td>NZ_AXWP01000091.1</td><td align="center">0.56435</td></tr><tr >
<td>NZ_AXWP01000092.1</td><td align="center">0.63184</td></tr><tr >
<td>NZ_AXWP01000093.1</td><td align="center">0.61155</td></tr><tr >
<td>NZ_AXWP01000094.1</td><td align="center">0.67327</td></tr><tr >
<td>NZ_AXWP01000095.1</td><td align="center">0.63269</td></tr><tr >
<td>NZ_AXWP01000096.1</td><td align="center">0.63499</td></tr><tr >
<td>NZ_AXWP01000097.1</td><td align="center">NA</td></tr><tr >
<td>NZ_AXWP01000098.1</td><td align="center">0.61849</td></tr><tr >
<td>NZ_AXWP01000099.1</td><td align="center">0.65675</td></tr><tr >
<td>NZ_AXWP01000100.1</td><td align="center">0.60409</td></tr><tr >
<td>NZ_AXWP01000101.1</td><td align="center">0.64605</td></tr><tr >
<td>NZ_AXWP01000102.1</td><td align="center">0.65162</td></tr><tr >
<td>NZ_AXWP01000103.1</td><td align="center">0.64887</td></tr><tr >
<td>NZ_AXWP01000104.1</td><td align="center">0.64780</td></tr><tr >
<td>NZ_AXWP01000105.1</td><td align="center">0.34789</td></tr><tr >
<td>NZ_AXWP01000106.1</td><td align="center">NA</td></tr><tr >
<td>NZ_AXWP01000107.1</td><td align="center">NA</td></tr><tr >
<td>NZ_AXWP01000108.1</td><td align="center">NA</td></tr><tr >
<td>NZ_AXWP01000109.1</td><td align="center">0.36475</td></tr><tr >
<td>NZ_AXWP01000110.1</td><td align="center">0.66148</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.4.1</td></tr>
<tr><td>R</td><td>4.3.1</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>12:18:36&nbsp;24&nbsp;Jul&nbsp;2025</td><td>12:21:46&nbsp;24&nbsp;Jul&nbsp;2025</td><td>3&nbsp;minutes&nbsp;10&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>12:21:46&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:18:00&nbsp;24&nbsp;Jul&nbsp;2025</td><td>56&nbsp;minutes&nbsp;14&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>13:18:00&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:20:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>2&nbsp;minutes&nbsp;48&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>13:20:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:29:08&nbsp;24&nbsp;Jul&nbsp;2025</td><td>8&nbsp;minutes&nbsp;20&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>13:29:08&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:39:26&nbsp;24&nbsp;Jul&nbsp;2025</td><td>10&nbsp;minutes&nbsp;18&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>13:39:26&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:55:44&nbsp;24&nbsp;Jul&nbsp;2025</td><td>16&nbsp;minutes&nbsp;18&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>13:55:44&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:00:16&nbsp;24&nbsp;Jul&nbsp;2025</td><td>4&nbsp;minutes&nbsp;32&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>14:00:16&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:06:32&nbsp;24&nbsp;Jul&nbsp;2025</td><td>6&nbsp;minutes&nbsp;16&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>14:06:32&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:10:53&nbsp;24&nbsp;Jul&nbsp;2025</td><td>4&nbsp;minutes&nbsp;21&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>14:10:53&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:11:36&nbsp;24&nbsp;Jul&nbsp;2025</td><td>43&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>14:11:36&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:47:44&nbsp;24&nbsp;Jul&nbsp;2025</td><td>36&nbsp;minutes&nbsp;8&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>14:47:44&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:47:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>4&nbsp;seconds</td></tr>
<tr><td>Output</td><td>14:47:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:50:23&nbsp;24&nbsp;Jul&nbsp;2025</td><td>2&nbsp;minutes&nbsp;35&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>14:47:48&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:48:00&nbsp;24&nbsp;Jul&nbsp;2025</td><td>12&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>14:48:00&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:48:00&nbsp;24&nbsp;Jul&nbsp;2025</td><td>0&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>2&nbsp;hours&nbsp;31&nbsp;minutes&nbsp;59&nbsp;seconds</b></td>
</tr>
</table></body></html>