<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.36.0&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/B9NRXP_14_S118_R1_001.error_rates.pdf">errors</a></td><td>B9NRXP_14_S118_R1_001</td><td align="right">4,343,093</td><td align="right">690,551,787</td><td align="right">100.0%</td><td align="right">159.0&nbsp;bases</td><td align="right">159&nbsp;bases</td><td align="right">96.3%</td></tr><tr ><td><a href="calibration/B9NRXP_14_S118_R2_001.error_rates.pdf">errors</a></td><td>B9NRXP_14_S118_R2_001</td><td align="right">4,321,988</td><td align="right">687,196,092</td><td align="right">99.5%</td><td align="right">159.0&nbsp;bases</td><td align="right">159&nbsp;bases</td><td align="right">94.4%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>8,665,081</b></td><td align="right"><b>1,377,747,879</b></td><td align="right">99.7%</td><td align="right">159.0&nbsp;bases</td><td align="right"><b>159</b>&nbsp;bases</td><td align="right">95.3%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit dispersion</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/G533DRAFT_scaffold00001.1574834.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00001.1574834</td><td align="right">574,834</td><td align="center">262.8</td><td align="center">6.5</td><td align="center">11.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00002.2319789.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00002.2319789</td><td align="right">319,789</td><td align="center">267.4</td><td align="center">8.3</td><td align="center">6.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00003.3301651.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00003.3301651</td><td align="right">301,651</td><td align="center">292.2</td><td align="center">9.9</td><td align="center">6.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00004.4293788.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00004.4293788</td><td align="right">293,788</td><td align="center">252.4</td><td align="center">7.1</td><td align="center">5.7%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00005.5274622.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00005.5274622</td><td align="right">274,622</td><td align="center">249.8</td><td align="center">5.6</td><td align="center">5.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00006.6243144.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00006.6243144</td><td align="right">243,144</td><td align="center">296.0</td><td align="center">8.6</td><td align="center">5.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00007.7213901.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00007.7213901</td><td align="right">213,901</td><td align="center">315.4</td><td align="center">9.2</td><td align="center">5.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00008.8212643.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00008.8212643</td><td align="right">212,643</td><td align="center">350.3</td><td align="center">10.3</td><td align="center">5.7%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00009.9192341.overview.png">coverage</a></td><td><a href="calibration/8.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00009.9192341</td><td align="right">192,341</td><td align="center">296.7</td><td align="center">7.2</td><td align="center">4.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00010.10180238.overview.png">coverage</a></td><td><a href="calibration/9.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00010.10180238</td><td align="right">180,238</td><td align="center">330.7</td><td align="center">6.6</td><td align="center">4.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00011.11154224.overview.png">coverage</a></td><td><a href="calibration/10.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00011.11154224</td><td align="right">154,224</td><td align="center">313.3</td><td align="center">7.6</td><td align="center">3.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00012.12145687.overview.png">coverage</a></td><td><a href="calibration/11.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00012.12145687</td><td align="right">145,687</td><td align="center">307.0</td><td align="center">7.3</td><td align="center">3.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00013.13123493.overview.png">coverage</a></td><td><a href="calibration/12.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00013.13123493</td><td align="right">123,493</td><td align="center">326.4</td><td align="center">7.5</td><td align="center">3.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00014.14113076.overview.png">coverage</a></td><td><a href="calibration/13.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00014.14113076</td><td align="right">113,076</td><td align="center">260.3</td><td align="center">6.3</td><td align="center">2.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00015.1587970.overview.png">coverage</a></td><td><a href="calibration/14.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00015.1587970</td><td align="right">87,970</td><td align="center">292.6</td><td align="center">6.1</td><td align="center">1.9%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00016.1686454.overview.png">coverage</a></td><td><a href="calibration/15.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00016.1686454</td><td align="right">86,454</td><td align="center">298.0</td><td align="center">7.7</td><td align="center">1.9%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00017.1785618.overview.png">coverage</a></td><td><a href="calibration/16.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00017.1785618</td><td align="right">85,618</td><td align="center">290.9</td><td align="center">8.4</td><td align="center">2.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00018.1853640.overview.png">coverage</a></td><td><a href="calibration/17.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00018.1853640</td><td align="right">53,640</td><td align="center">305.8</td><td align="center">7.4</td><td align="center">1.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00019.1952962.overview.png">coverage</a></td><td><a href="calibration/18.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00019.1952962</td><td align="right">52,962</td><td align="center">305.6</td><td align="center">7.0</td><td align="center">1.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00020.2041924.overview.png">coverage</a></td><td><a href="calibration/19.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00020.2041924</td><td align="right">41,924</td><td align="center">495.1</td><td align="center">29.7</td><td align="center">1.5%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00021.2141489.overview.png">coverage</a></td><td><a href="calibration/20.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00021.2141489</td><td align="right">41,489</td><td align="center">255.2</td><td align="center">6.1</td><td align="center">0.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00022.2237030.overview.png">coverage</a></td><td><a href="calibration/21.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00022.2237030</td><td align="right">37,030</td><td align="center">501.2</td><td align="center">12.4</td><td align="center">1.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00023.2335948.overview.png">coverage</a></td><td><a href="calibration/22.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00023.2335948</td><td align="right">35,948</td><td align="center">465.6</td><td align="center">14.1</td><td align="center">1.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00024.2435397.overview.png">coverage</a></td><td><a href="calibration/23.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00024.2435397</td><td align="right">35,397</td><td align="center">356.7</td><td align="center">8.7</td><td align="center">1.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00025.2531048.overview.png">coverage</a></td><td><a href="calibration/24.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00025.2531048</td><td align="right">31,048</td><td align="center">484.3</td><td align="center">16.5</td><td align="center">1.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00026.2622708.overview.png">coverage</a></td><td><a href="calibration/25.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00026.2622708</td><td align="right">22,708</td><td align="center">474.2</td><td align="center">18.6</td><td align="center">0.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00027.2720965.overview.png">coverage</a></td><td><a href="calibration/26.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00027.2720965</td><td align="right">20,965</td><td align="center">519.8</td><td align="center">25.8</td><td align="center">0.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00028.2820332.overview.png">coverage</a></td><td><a href="calibration/27.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00028.2820332</td><td align="right">20,332</td><td align="center">483.4</td><td align="center">13.2</td><td align="center">0.8%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00029.2920209.overview.png">coverage</a></td><td><a href="calibration/28.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00029.2920209</td><td align="right">20,209</td><td align="center">398.2</td><td align="center">4.4</td><td align="center">0.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00030.3018396.overview.png">coverage</a></td><td><a href="calibration/29.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00030.3018396</td><td align="right">18,396</td><td align="center">314.4</td><td align="center">5.4</td><td align="center">0.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00031.3116827.overview.png">coverage</a></td><td><a href="calibration/30.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00031.3116827</td><td align="right">16,827</td><td align="center">333.5</td><td align="center">4.2</td><td align="center">0.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00032.3215758.overview.png">coverage</a></td><td><a href="calibration/31.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00032.3215758</td><td align="right">15,758</td><td align="center">349.6</td><td align="center">6.1</td><td align="center">0.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00033.3310486.overview.png">coverage</a></td><td><a href="calibration/32.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00033.3310486</td><td align="right">10,486</td><td align="center">352.7</td><td align="center">4.3</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00034.3410436.overview.png">coverage</a></td><td><a href="calibration/33.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00034.3410436</td><td align="right">10,436</td><td align="center">259.8</td><td align="center">4.6</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00035.3510388.overview.png">coverage</a></td><td><a href="calibration/34.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00035.3510388</td><td align="right">10,388</td><td align="center">561.9</td><td align="center">32.6</td><td align="center">0.6%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00036.3610198.overview.png">coverage</a></td><td><a href="calibration/35.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00036.3610198</td><td align="right">10,198</td><td align="center">471.4</td><td align="center">5.5</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00037.379347.overview.png">coverage</a></td><td><a href="calibration/36.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00037.379347</td><td align="right">9,347</td><td align="center">3970.8</td><td align="center">23100.0</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00038.389328.overview.png">coverage</a></td><td><a href="calibration/37.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00038.389328</td><td align="right">9,328</td><td align="center">476.0</td><td align="center">2.7</td><td align="center">0.4%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00039.399189.overview.png">coverage</a></td><td><a href="calibration/38.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00039.399189</td><td align="right">9,189</td><td align="center">315.3</td><td align="center">7.2</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00040.408025.overview.png">coverage</a></td><td><a href="calibration/39.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00040.408025</td><td align="right">8,025</td><td align="center">360.8</td><td align="center">4.3</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00041.417912.overview.png">coverage</a></td><td><a href="calibration/40.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00041.417912</td><td align="right">7,912</td><td align="center">341.7</td><td align="center">9.4</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00042.427595.overview.png">coverage</a></td><td><a href="calibration/41.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00042.427595</td><td align="right">7,595</td><td align="center">566.4</td><td align="center">19.4</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00043.436536.overview.png">coverage</a></td><td><a href="calibration/42.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00043.436536</td><td align="right">6,536</td><td align="center">295.8</td><td align="center">4.3</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00044.446528.overview.png">coverage</a></td><td><a href="calibration/43.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00044.446528</td><td align="right">6,528</td><td align="center">590.9</td><td align="center">6.7</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00045.455643.overview.png">coverage</a></td><td><a href="calibration/44.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00045.455643</td><td align="right">5,643</td><td align="center">761.8</td><td align="center">6.7</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00046.465641.overview.png">coverage</a></td><td><a href="calibration/45.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00046.465641</td><td align="right">5,641</td><td align="center">570.5</td><td align="center">2.5</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00047.475562.overview.png">coverage</a></td><td><a href="calibration/46.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00047.475562</td><td align="right">5,562</td><td align="center">289.3</td><td align="center">20.8</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00048.485241.overview.png">coverage</a></td><td><a href="calibration/47.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00048.485241</td><td align="right">5,241</td><td align="center">513.0</td><td align="center">2.6</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00049.494928.overview.png">coverage</a></td><td><a href="calibration/48.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00049.494928</td><td align="right">4,928</td><td align="center">569.3</td><td align="center">10.2</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00050.504879.overview.png">coverage</a></td><td><a href="calibration/49.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00050.504879</td><td align="right">4,879</td><td align="center">557.2</td><td align="center">6.8</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00051.514248.overview.png">coverage</a></td><td><a href="calibration/50.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00051.514248</td><td align="right">4,248</td><td align="center">552.9</td><td align="center">12.0</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00052.523303.overview.png">coverage</a></td><td><a href="calibration/51.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00052.523303</td><td align="right">3,303</td><td align="center">598.5</td><td align="center">4.7</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00053.533012.overview.png">coverage</a></td><td><a href="calibration/52.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00053.533012</td><td align="right">3,012</td><td align="center">383.6</td><td align="center">3.5</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00054.542899.overview.png">coverage</a></td><td><a href="calibration/53.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00054.542899</td><td align="right">2,899</td><td align="center">274.9</td><td align="center">4.6</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00055.552851.overview.png">coverage</a></td><td><a href="calibration/54.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00055.552851</td><td align="right">2,851</td><td align="center">571.8</td><td align="center">7.6</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00056.562677.overview.png">coverage</a></td><td><a href="calibration/55.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00056.562677</td><td align="right">2,677</td><td align="center">1676.5</td><td align="center">34.8</td><td align="center">0.3%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00057.572543.overview.png">coverage</a></td><td><a href="calibration/56.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00057.572543</td><td align="right">2,543</td><td align="center">534.3</td><td align="center">13.5</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00058.582523.overview.png">coverage</a></td><td><a href="calibration/57.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00058.582523</td><td align="right">2,523</td><td align="center">339.1</td><td align="center">3.3</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00059.592504.overview.png">coverage</a></td><td><a href="calibration/58.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00059.592504</td><td align="right">2,504</td><td align="center">347.4</td><td align="center">2.3</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00060.602354.overview.png">coverage</a></td><td><a href="calibration/59.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00060.602354</td><td align="right">2,354</td><td align="center">375.3</td><td align="center">3.6</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00061.612070.overview.png">coverage</a></td><td><a href="calibration/60.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00061.612070</td><td align="right">2,070</td><td align="center">428.5</td><td align="center">5.7</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00062.621919.overview.png">coverage</a></td><td><a href="calibration/61.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00062.621919</td><td align="right">1,919</td><td align="center">1069.8</td><td align="center">17.0</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00063.631836.overview.png">coverage</a></td><td><a href="calibration/62.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00063.631836</td><td align="right">1,836</td><td align="center">401.1</td><td align="center">5.1</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00064.641745.overview.png">coverage</a></td><td><a href="calibration/63.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00064.641745</td><td align="right">1,745</td><td align="center">634.5</td><td align="center">6.4</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00065.651652.overview.png">coverage</a></td><td><a href="calibration/64.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00065.651652</td><td align="right">1,652</td><td align="center">1415.3</td><td align="center">9.5</td><td align="center">0.2%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00066.661607.overview.png">coverage</a></td><td><a href="calibration/65.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00066.661607</td><td align="right">1,607</td><td align="center">460.6</td><td align="center">29.7</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00067.671453.overview.png">coverage</a></td><td><a href="calibration/66.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00067.671453</td><td align="right">1,453</td><td align="center">477.8</td><td align="center">3.0</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr class="gray_table_row"><td><a href="evidence/G533DRAFT_scaffold00068.681424.overview.png">coverage</a></td><td><a href="calibration/67.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00068.681424</td><td align="right">1,424</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00069.691416.overview.png">coverage</a></td><td><a href="calibration/68.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00069.691416</td><td align="right">1,416</td><td align="center">124.0</td><td align="center">16.9</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00070.701391.overview.png">coverage</a></td><td><a href="calibration/69.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00070.701391</td><td align="right">1,391</td><td align="center">654.7</td><td align="center">6.7</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00071.711385.overview.png">coverage</a></td><td><a href="calibration/70.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00071.711385</td><td align="right">1,385</td><td align="center">399.6</td><td align="center">3.9</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00072.721370.overview.png">coverage</a></td><td><a href="calibration/71.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00072.721370</td><td align="right">1,370</td><td align="center">522.9</td><td align="center">5.3</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00073.731307.overview.png">coverage</a></td><td><a href="calibration/72.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00073.731307</td><td align="right">1,307</td><td align="center">637.8</td><td align="center">4.3</td><td align="center">0.1%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00074.741203.overview.png">coverage</a></td><td><a href="calibration/73.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00074.741203</td><td align="right">1,203</td><td align="center">476.3</td><td align="center">4.6</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00075.751031.overview.png">coverage</a></td><td><a href="calibration/74.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00075.751031</td><td align="right">1,031</td><td align="center">440.0</td><td align="center">4.5</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr><td><a href="evidence/G533DRAFT_scaffold00076.761031.overview.png">coverage</a></td><td><a href="calibration/75.unique_coverage.pdf">distribution</a></td><td>G533DRAFT_scaffold00076.761031</td><td align="right">1,031</td><td align="center">434.4</td><td align="center">9.7</td><td align="center">0.0%</td><td>Genus species strain strain.</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>4,258,722</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit dispersion</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<p><span class="gray_table_row">Insufficient coverage</span> Reference sequence counted as entirely deleted due to low coverage. Try either the <code>-t,--targeted-sequencing</code> or the <code>-c,--contig-reference </code> option if you want mutations called for these reference sequences.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">38335</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;2</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">561</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.043</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>G533DRAFT_scaffold00001.1574834</td><td align="center">0.71540</td></tr><tr >
<td>G533DRAFT_scaffold00002.2319789</td><td align="center">0.71053</td></tr><tr >
<td>G533DRAFT_scaffold00003.3301651</td><td align="center">0.69735</td></tr><tr >
<td>G533DRAFT_scaffold00004.4293788</td><td align="center">0.71957</td></tr><tr >
<td>G533DRAFT_scaffold00005.5274622</td><td align="center">0.72361</td></tr><tr >
<td>G533DRAFT_scaffold00006.6243144</td><td align="center">0.69785</td></tr><tr >
<td>G533DRAFT_scaffold00007.7213901</td><td align="center">0.68575</td></tr><tr >
<td>G533DRAFT_scaffold00008.8212643</td><td align="center">0.67448</td></tr><tr >
<td>G533DRAFT_scaffold00009.9192341</td><td align="center">0.69561</td></tr><tr >
<td>G533DRAFT_scaffold00010.10180238</td><td align="center">0.68321</td></tr><tr >
<td>G533DRAFT_scaffold00011.11154224</td><td align="center">0.68728</td></tr><tr >
<td>G533DRAFT_scaffold00012.12145687</td><td align="center">0.69147</td></tr><tr >
<td>G533DRAFT_scaffold00013.13123493</td><td align="center">0.68066</td></tr><tr >
<td>G533DRAFT_scaffold00014.14113076</td><td align="center">0.71454</td></tr><tr >
<td>G533DRAFT_scaffold00015.1587970</td><td align="center">0.70410</td></tr><tr >
<td>G533DRAFT_scaffold00016.1686454</td><td align="center">0.70231</td></tr><tr >
<td>G533DRAFT_scaffold00017.1785618</td><td align="center">0.69882</td></tr><tr >
<td>G533DRAFT_scaffold00018.1853640</td><td align="center">0.68410</td></tr><tr >
<td>G533DRAFT_scaffold00019.1952962</td><td align="center">0.69336</td></tr><tr >
<td>G533DRAFT_scaffold00020.2041924</td><td align="center">0.62783</td></tr><tr >
<td>G533DRAFT_scaffold00021.2141489</td><td align="center">0.72246</td></tr><tr >
<td>G533DRAFT_scaffold00022.2237030</td><td align="center">0.61526</td></tr><tr >
<td>G533DRAFT_scaffold00023.2335948</td><td align="center">0.62995</td></tr><tr >
<td>G533DRAFT_scaffold00024.2435397</td><td align="center">0.66743</td></tr><tr >
<td>G533DRAFT_scaffold00025.2531048</td><td align="center">0.61865</td></tr><tr >
<td>G533DRAFT_scaffold00026.2622708</td><td align="center">0.62609</td></tr><tr >
<td>G533DRAFT_scaffold00027.2720965</td><td align="center">0.61023</td></tr><tr >
<td>G533DRAFT_scaffold00028.2820332</td><td align="center">0.62861</td></tr><tr >
<td>G533DRAFT_scaffold00029.2920209</td><td align="center">0.64872</td></tr><tr >
<td>G533DRAFT_scaffold00030.3018396</td><td align="center">0.69356</td></tr><tr >
<td>G533DRAFT_scaffold00031.3116827</td><td align="center">0.68053</td></tr><tr >
<td>G533DRAFT_scaffold00032.3215758</td><td align="center">0.66709</td></tr><tr >
<td>G533DRAFT_scaffold00033.3310486</td><td align="center">0.67457</td></tr><tr >
<td>G533DRAFT_scaffold00034.3410436</td><td align="center">0.71245</td></tr><tr >
<td>G533DRAFT_scaffold00035.3510388</td><td align="center">0.58799</td></tr><tr >
<td>G533DRAFT_scaffold00036.3610198</td><td align="center">0.66566</td></tr><tr >
<td>G533DRAFT_scaffold00037.379347</td><td align="center">1.00000</td></tr><tr >
<td>G533DRAFT_scaffold00038.389328</td><td align="center">0.61125</td></tr><tr >
<td>G533DRAFT_scaffold00039.399189</td><td align="center">0.68981</td></tr><tr >
<td>G533DRAFT_scaffold00040.408025</td><td align="center">0.66260</td></tr><tr >
<td>G533DRAFT_scaffold00041.417912</td><td align="center">0.68074</td></tr><tr >
<td>G533DRAFT_scaffold00042.427595</td><td align="center">0.60074</td></tr><tr >
<td>G533DRAFT_scaffold00043.436536</td><td align="center">0.70385</td></tr><tr >
<td>G533DRAFT_scaffold00044.446528</td><td align="center">0.58818</td></tr><tr >
<td>G533DRAFT_scaffold00045.455643</td><td align="center">0.53515</td></tr><tr >
<td>G533DRAFT_scaffold00046.465641</td><td align="center">0.59141</td></tr><tr >
<td>G533DRAFT_scaffold00047.475562</td><td align="center">0.69969</td></tr><tr >
<td>G533DRAFT_scaffold00048.485241</td><td align="center">0.60492</td></tr><tr >
<td>G533DRAFT_scaffold00049.494928</td><td align="center">0.59256</td></tr><tr >
<td>G533DRAFT_scaffold00050.504879</td><td align="center">0.58982</td></tr><tr >
<td>G533DRAFT_scaffold00051.514248</td><td align="center">0.60135</td></tr><tr >
<td>G533DRAFT_scaffold00052.523303</td><td align="center">0.60263</td></tr><tr >
<td>G533DRAFT_scaffold00053.533012</td><td align="center">0.67928</td></tr><tr >
<td>G533DRAFT_scaffold00054.542899</td><td align="center">0.73456</td></tr><tr >
<td>G533DRAFT_scaffold00055.552851</td><td align="center">0.61364</td></tr><tr >
<td>G533DRAFT_scaffold00056.562677</td><td align="center">0.40335</td></tr><tr >
<td>G533DRAFT_scaffold00057.572543</td><td align="center">0.64137</td></tr><tr >
<td>G533DRAFT_scaffold00058.582523</td><td align="center">0.68549</td></tr><tr >
<td>G533DRAFT_scaffold00059.592504</td><td align="center">0.69449</td></tr><tr >
<td>G533DRAFT_scaffold00060.602354</td><td align="center">0.67035</td></tr><tr >
<td>G533DRAFT_scaffold00061.612070</td><td align="center">0.66208</td></tr><tr >
<td>G533DRAFT_scaffold00062.621919</td><td align="center">0.53517</td></tr><tr >
<td>G533DRAFT_scaffold00063.631836</td><td align="center">0.64135</td></tr><tr >
<td>G533DRAFT_scaffold00064.641745</td><td align="center">0.56476</td></tr><tr >
<td>G533DRAFT_scaffold00065.651652</td><td align="center">0.47942</td></tr><tr >
<td>G533DRAFT_scaffold00066.661607</td><td align="center">0.64624</td></tr><tr >
<td>G533DRAFT_scaffold00067.671453</td><td align="center">0.66930</td></tr><tr >
<td>G533DRAFT_scaffold00068.681424</td><td align="center">NA</td></tr><tr >
<td>G533DRAFT_scaffold00069.691416</td><td align="center">1.00000</td></tr><tr >
<td>G533DRAFT_scaffold00070.701391</td><td align="center">0.62257</td></tr><tr >
<td>G533DRAFT_scaffold00071.711385</td><td align="center">0.69892</td></tr><tr >
<td>G533DRAFT_scaffold00072.721370</td><td align="center">0.63467</td></tr><tr >
<td>G533DRAFT_scaffold00073.731307</td><td align="center">0.59487</td></tr><tr >
<td>G533DRAFT_scaffold00074.741203</td><td align="center">0.65835</td></tr><tr >
<td>G533DRAFT_scaffold00075.751031</td><td align="center">0.62715</td></tr><tr >
<td>G533DRAFT_scaffold00076.761031</td><td align="center">0.63975</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.4.1</td></tr>
<tr><td>R</td><td>4.3.1</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>13:57:15&nbsp;24&nbsp;Jul&nbsp;2025</td><td>13:59:06&nbsp;24&nbsp;Jul&nbsp;2025</td><td>1&nbsp;minute&nbsp;51&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>13:59:06&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:31:37&nbsp;24&nbsp;Jul&nbsp;2025</td><td>32&nbsp;minutes&nbsp;31&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>14:31:37&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:33:18&nbsp;24&nbsp;Jul&nbsp;2025</td><td>1&nbsp;minute&nbsp;41&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>14:33:18&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:39:06&nbsp;24&nbsp;Jul&nbsp;2025</td><td>5&nbsp;minutes&nbsp;48&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>14:39:06&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:39:44&nbsp;24&nbsp;Jul&nbsp;2025</td><td>38&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>14:39:44&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:48:30&nbsp;24&nbsp;Jul&nbsp;2025</td><td>8&nbsp;minutes&nbsp;46&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>14:48:30&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:51:05&nbsp;24&nbsp;Jul&nbsp;2025</td><td>2&nbsp;minutes&nbsp;35&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>14:51:05&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:55:27&nbsp;24&nbsp;Jul&nbsp;2025</td><td>4&nbsp;minutes&nbsp;22&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>14:55:27&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:57:46&nbsp;24&nbsp;Jul&nbsp;2025</td><td>2&nbsp;minutes&nbsp;19&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>14:57:46&nbsp;24&nbsp;Jul&nbsp;2025</td><td>14:58:27&nbsp;24&nbsp;Jul&nbsp;2025</td><td>41&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>14:58:27&nbsp;24&nbsp;Jul&nbsp;2025</td><td>15:20:30&nbsp;24&nbsp;Jul&nbsp;2025</td><td>22&nbsp;minutes&nbsp;3&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>15:20:30&nbsp;24&nbsp;Jul&nbsp;2025</td><td>15:20:32&nbsp;24&nbsp;Jul&nbsp;2025</td><td>2&nbsp;seconds</td></tr>
<tr><td>Output</td><td>15:20:32&nbsp;24&nbsp;Jul&nbsp;2025</td><td>15:21:58&nbsp;24&nbsp;Jul&nbsp;2025</td><td>1&nbsp;minute&nbsp;26&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>15:20:32&nbsp;24&nbsp;Jul&nbsp;2025</td><td>15:20:38&nbsp;24&nbsp;Jul&nbsp;2025</td><td>6&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>15:20:38&nbsp;24&nbsp;Jul&nbsp;2025</td><td>15:20:38&nbsp;24&nbsp;Jul&nbsp;2025</td><td>0&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>1&nbsp;hour&nbsp;24&nbsp;minutes&nbsp;49&nbsp;seconds</b></td>
</tr>
</table></body></html>